### ISSUE TITLE
Nativescript 8 - TypeError: Cannot read property 'high' of undefined

### ISSUE THREAD

**Issue by jnorkus** [farfromrefug/nativescript-gps#15]
In Nativescript 8 the `Accuracy` enum is now accessed via `CoreTypes`, by importing it from `@nativescript/core`.

Because of that I am getting this error.
```
JS: TypeError: Cannot read property 'high' of undefined
JS:     at criteriaFromOptions (file: node_modules/@nativescript-community/gps/gps.android.js:154:54)
```

I am happy to submit a PR.

**Comment by farfromrefug** [comment]
@jnorkus you can i ll publish it as major version change

**Comment by jnorkus** [comment]
@farfromrefug looks like there is already a PR: https://github.com/nativescript-community/gps/pull/14

**Comment by farfromrefug** [comment]
@jnorkus  published as 3.0.0
